src/btoropt/program.py

```diff
@@ -119,8 +120,9 @@ def serialize(self) -> str:
 
 
 class Output(Instruction):
-    def __init__(self, lid: int, out: Instruction):
+    def __init__(self, lid: int, out: Instruction, name: str):
```

Comment:
Oh nice, I didn't know outputs could be named (I don't typically use them as they aren't supported by model checkers), neat!